apps/web/src/pages/ItTrainingFundPage/ItTrainingFundPage.tsx

```diff
@@ -0,0 +1,578 @@
+import { defineMessage, useIntl } from "react-intl";
+import MapIcon from "@heroicons/react/24/outline/MapIcon";
+import BookmarkSquareIcon from "@heroicons/react/24/outline/BookmarkSquareIcon";
+import LightBulbIcon from "@heroicons/react/24/outline/LightBulbIcon";
+import { ReactNode } from "react";
+
+import { CardBasic, CardFlat, Heading, Link } from "@gc-digital-talent/ui";
+import { getLocale } from "@gc-digital-talent/i18n";
+
+import Hero from "~/components/HeroDeprecated";
+import useBreadcrumbs from "~/hooks/useBreadcrumbs";
+import useRoutes from "~/hooks/useRoutes";
+import image1 from "~/assets/img/it-training-fund-1.webp";
+import image2 from "~/assets/img/it-training-fund-2.webp";
+import image3 from "~/assets/img/it-training-fund-3.webp";
+
+const externalLinkAccessor = (href: string, chunks: ReactNode) => {
+  return (
+    <Link href={href} color="secondary" external>
+      <strong>{chunks}</strong>
+    </Link>
+  );
+};
+
+const pageTitle = defineMessage({
+  defaultMessage: "IT Community Training and Development Fund",
+  id: "wOITol",
+  description: "page title for the IT training fund page",
+});
+
+const pageSubtitle = defineMessage({
+  defaultMessage:
+    "Explore learning opportunities for IT employees to build and strengthen skills.",
+  id: "A94Zgi",
+  description: "Page subtitle for the IT training fund page",
+});
+
+export const Component = () => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+
+  const signUpUrl = {
+    en: "https://forms-formulaires.alpha.canada.ca/en/id/cm2f6uhix006gd1ecypq52aoj",
+    fr: "https://forms-formulaires.alpha.canada.ca/fr/id/cm2f6uhix006gd1ecypq52aoj",
+  } as const;
+
+  const crumbs = useBreadcrumbs({
+    crumbs: [
+      {
+        label: intl.formatMessage(pageTitle),
+        url: paths.itTrainingFund(),
+      },
+    ],
+  });
+
+  return (
+    <>
+      <Hero
+        title={intl.formatMessage(pageTitle)}
+        subtitle={intl.formatMessage(pageSubtitle)}
+        crumbs={crumbs}
+        centered
+      />
+      <div data-h2-padding="base(x3, 0)">
+        <div data-h2-wrapper="base(center, large, x1) p-tablet(center, large, x2)">
+          <div
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column)"
+            data-h2-gap="base(x3)"
+          >
+            {/* Investing in the future of IT talent */}
+            <div>
+              <Heading
+                Icon={MapIcon}
+                size="h2"
```

Comment:
Good noticing!  Yeah, this came up during the design review, too. (And the design review of my admin styling PR :disappointed: )  It's our standard component heading that is off.  If you check Figma you can see it is supposed to be h2 sized.
![image](https://github.com/user-attachments/assets/a0f18cb3-0be6-4896-8c6a-dd5a82843c0e)
If I fudge it to h3 now it will just be off again when we fix the component.  I've gone ahead and started the issue to realign the component:
https://github.com/GCTC-NTGC/gc-digital-talent/issues/11859

Reply:
I think I mentioned that below. I remember talking about it with Eric, and I think he made an issue.